src/base/rss/rssfeed.cpp

```diff
@@ -86,6 +86,10 @@ Feed::Feed(const QString &url, Manager *manager)
     refresh();
 }
 
+Feed::Feed(const QString &url)
+    : Feed(url, nullptr)
+{}
```

Comment:
per 1a, empty bodies are {}. Does that not apply to constructors? or only apply when on a single line?

Not trying to argue, just asking.